src/api/api.js

```diff
@@ -1153,11 +1153,11 @@ export {
   getUserModelInfoApi,
   getBookByIdApi,
   getBookRecommendListApi,
```

Comment:
maybe should be "Recommended" instead of "Recommend" ? 
getBookRecommendedListApi()